real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/robots/TransistorAuton.java

```diff
@@ -67,7 +67,7 @@ private TransistorAuton() {
         nodeList.add(new RBSMNode(NodeChannel.ENCODER, NodeChannel.STEERING, RobobuggyConfigFile.getComPortRBSM(),
                 RobobuggyConfigFile.RBSM_COMMAND_PERIOD));
 //		nodeList.add(new CameraNode(NodeChannel.PUSHBAR_CAMERA, 100));
-        nodeList.add(new HillCrestImuNode());
+        nodeList.add(new HillCrestImuNode()); //Code doesn't work
```

Comment:
Please don't put comments like this in the repo. The code didn't work because the IMU wasn't connected